### ISSUE TITLE
The same error of #40 but with categorize

### ISSUE THREAD

**Issue by avurro** [ygimenez/Pagination-Utils#41]
**Checklist**

Make sure that you've checked all the items below.

- [X] Bot has the following permissions:
  - MESSAGE_ADD_REACTION
  - MESSAGE_EXT_EMOJI
  - MESSAGE_READ/WRITE
  - VIEW_CHANNEL
- [X] If using `JDABuilder.createLight()`, you added the following gateway intents:
    - GUILD_MESSAGES
    - GUILD_MESSAGE_REACTIONS
- [X] If using `.setRemoveOnReact(true)`, you have the following permission:
    - MESSAGE_MANAGE
- [X] PaginationUtils is up-to-date.
- [X] You have activated the library as descripted in the README.

**Describe the bug**
The same of #40 but for categorize

**To Reproduce**

See #40 

**Expected behavior**

See #40

**Comment by avurro** [comment]
The problem is here:
Pages.class
```java
810   ...
811   } else if (emoji != null && Objects.equals(emoji, currCat)) {
812   ...
```
I think it is enough to change it to:
```java
810   ...
811   } else if (emoji != null && (currCat == null || !Objects.equals(emoji, currCat))) {
812   ...
```

**Comment by ygimenez** [comment]
`Objects.equals()` already considers null values as false, I just need to follow the debugger to see where it's falling. In the past case it was because it was comparing the Emote with the wrong collection, so it might be something similar in this case.

However, since it's very late where I live I'll only be able to solve it tomorrow.

**Comment by avurro** [comment]
In this case the null value must be evaluated as **true** and the adjacent condition negated.

this is the condition that indicates whether to change content or not. The current page must be different from the clicked one and not the same (if I understand correctly)

**Comment by ygimenez** [comment]
Issue fixed in [release 3.0.8](https://github.com/ygimenez/Pagination-Utils/tree/3.0.8)

**Comment by avurro** [comment]
thanks 👍
